MekHQ/unittests/mekhq/campaign/parts/equipment/EquipmentPartTest.java

```diff
@@ -1449,8 +1449,7 @@ public void checkWeaponBayOnlyWeaponRemovedTest() {
         when(weaponBay.getLocation()).thenReturn(location);
         when(weaponBay.getType()).thenReturn(bayWeaponType);
         List<WeaponMounted> bayWeapons = new ArrayList<>();
-        //bayWeapons.addElement(33);
-        bayWeapons.add(weaponBay);
+        bayWeapons.add(mounted);
```

Comment:
Not sure what 33 was supposed to be as this equipment number is not used in this test and its supposed to be testing a weapon bay with only one weapon.